src/components/views/rooms/SendMessageComposer.js

```diff
@@ -88,6 +90,25 @@ export function createMessageContent(model, permalinkCreator, replyToEvent) {
     return content;
 }
 
+// exported for tests
+export function isQuickReaction(model) {
+    const parts = model.parts;
+    if (parts.length == 0) return false;
+    let text = parts[0].text;
+    text += parts[1] ? parts[1].text : "";
+    // shortcut takes the form "+:emoji:" or "+ :emoji:""
+    // can be in 1 or 2 parts
+    if (parts.length <= 2) {
+        const hasShortcut = text.startsWith("+") || text.startsWith("+ ");
```

Comment:
you could use `startsWith(model, str)` from `editor/serialize` here

Reply:
With the change below, is it okay if I keep this as is? `startsWith` from serialize seems like a good choice here, but the "+" character actually gets created as a PillCandidate here, causing the model startsWith to always return false. https://github.com/matrix-org/matrix-react-sdk/blob/a1159545c4c68c599f206bb43c14b7a74b29c8ff/src/editor/parts.ts#L465-L477

Reply:
Ah yes, good point :+1: